Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -56,9 +56,14 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
             var content = new Y.eZ.Content(),
                 version = new Y.eZ.Version(),
                 type = this.get('contentType'),
-                defaultFields = {};
+                defaultFields = {},
+                contentTypeNames,
+                contentTypeName;
 
-            content.set('name', 'New "' + this.get('contentType').get('names')['eng-GB'] + '"');
+            contentTypeNames = type.get('names');
+            contentTypeName = contentTypeNames[this.get('app').get('defaultLanguageCode')] || contentTypeNames[0];
```

Comment:
> I'm not following. Besides, still relevant? Given this is now marked as outdated.

My question was about how the priorizedLanguages info retrieved, I didn't find any suitable method for this and could have implemented a provider differently (not listing all siteaccesses).

Still relevant, I just commited a fix that resolves an issue in the content create page title being set to f.e. "New fre-FR" see [150fff4179878e1b](https://github.com/ezsystems/PlatformUIBundle/pull/581/commits/150fff4179878e1b)